src/config/rcxml.c

```diff
@@ -537,7 +538,10 @@ fill_libinput_category(char *nodename, char *content)
 	} else if (!strcasecmp(nodename, "leftHanded")) {
 		set_bool_as_int(content, &current_libinput_category->left_handed);
 	} else if (!strcasecmp(nodename, "pointerSpeed")) {
-		current_libinput_category->pointer_speed = atof(content);
+		double v;
+		if (parse_double(content, &v)) {
+			current_libinput_category->pointer_speed = v;
+		}
```

Comment:
What is the reason for checking the result here? I assume it is because of double -> float, should we create a small wrapper like `set_float()` instead?